niri-config/src/lib.rs

```diff
@@ -431,6 +433,43 @@ pub enum GradientRelativeTo {
     WorkspaceView,
 }
 
+#[derive(knuffel::Decode, Debug, Clone, Copy, PartialEq)]
+pub struct GradientInterpolation {
+    #[knuffel(property, default)]
+    pub color_space: GradientColorSpace,
+    #[knuffel(property, default)]
+    pub hue_interpol: HueInterpolation
+}
+
+#[derive(knuffel::DecodeScalar, Debug, Default, Clone, Copy, PartialEq, Eq)]
+pub enum GradientColorSpace {
+    #[default]
+    Srgb,
+    SrgbLinear,
+    DisplayP3,
+    A98Rgb,
+    ProphotoRgb,
+    Rec2020,
+    Lab,
+    Oklab,
+    Xyz,
+    XyzD50,
+    XyzD65,
+    Hsl,
+    Hwb,
+    Lch,
+    Oklch
```

Comment:
We don't need all of these here, just the ones we actually support. And I think it makes sense to support only srgb, srgb-linear, oklab and oklch, not sure any other would be useful in our setting?